src/config.rs

```diff
@@ -29,8 +29,8 @@ impl Default for Config {
         Config {
             source: "./".to_owned(),
             dest: "./".to_owned(),
-            layouts: "_layouts".to_owned(),
-            drafts: "_drafts".to_owned(),
+            templates: "templates".to_owned(),
+            drafts: "drafts".to_owned(),
```

Comment:
I think thats a good idea to split this into another issue.

> Personally, I think moving all build/cli-related files into a single .cobalt subdirectory allows us to add new tooling and features - like skeleton files - without having to worry about polluting the project space. It makes it clear what the user is responsible for maintaining within their project, and where they can go to modify how cobalt works in regards to their project.

Just to check, is your comment about a ".cobalt/" meant to be related to avoiding hiding files with a "_" prefix?